### ISSUE TITLE
deployed to heroku and content disappear 

### ISSUE THREAD

**Issue by avatarr** [meanjs/mean#1959]
I've deployed meanjs to heroku for fitst time but I can't see my content it's blank

![screenshot from 2018-04-01 21-00-35](https://user-images.githubusercontent.com/1640817/38173955-dfcaef88-35ef-11e8-9200-e2747daffe52.png)

URL: https://strawberry-shortcake-44338.herokuapp.com/
Any idea? 

Thanks in advance

**Comment by avatarr** [comment]
![screenshot from 2018-04-01 21-23-18](https://user-images.githubusercontent.com/1640817/38174174-9ea3f4ce-35f3-11e8-9ee1-e0c4eb1966c6.png)

work find in localhost with ```gulp prod```

**Comment by avatarr** [comment]
**Update**: I found the discussion on stack overflow they told let try to change NODE_ENV to development mode, You know it's worked the website is appeared and I figured it out the cause is ```resolve``` on angular route. I use ```resolve``` to load data on the back-end but I don't know why the UI is has been blocked like this but on the local it's work find. Any suggestion? 

Thanks in advanced.

**Comment by avatarr** [comment]
Anyone? I need help

**Comment by avatarr** [comment]
@lirantal

**Comment by dylib** [comment]
@avatarr: What happens if you try deploying with this version: https://github.com/Ghalleb/mean?

**Comment by mleanos** [comment]
The build is most likely failing on Heroku, due to the dev package dependencies not getting installed.

Please see these issues

https://github.com/meanjs/mean/issues/1411
#241

**Comment by UQONYX** [comment]
Any update on this?

**Comment by NSLog0** [comment]
No, I avoid using `resolve`. I find a solution to fix it but, no one talk about it. If anyone has experience about it, please tell me

**Comment by UQONYX** [comment]
I am using NODE_ENV="development" and everything works. However, I am also concerned about the same thing. The build clearly fails when put into production mode but works in development. I am not sure what the downsides are.

**Comment by NSLog0** [comment]
Actually, I stopped to develop a new project with MEAN.js because 
1. Don't get support for an issue immediately 
2. I go with React.js 
3. I had a few errors that couldn't solve. You can find the error that by filter my username
